src/deps.ts

```diff
@@ -7,3 +7,6 @@ export { readAll } from "https://deno.land/std@0.170.0/streams/read_all.ts";
 export { writeAllSync } from "https://deno.land/std@0.170.0/streams/write_all.ts";
 export { default as localDataDir } from "https://deno.land/x/dir@1.5.1/data_local_dir/mod.ts";
 export { RealEnvironment as DenoWhichRealEnvironment, which, whichSync } from "https://deno.land/x/which@0.2.1/mod.ts";
+export { default as stripAnsi } from "npm:strip-ansi@7.0.1";
+import dedentImport from "npm:string-dedent@3.0.1";
```

Comment:
We should find a deno.land/x implementation of this or implement it ourselves. I don't want to include non deno.land registries in dax. Also, npm specifiers are unstable in remote modules, so this would cause dax to require the --unstable flag at the moment.

Reply:
I've updated to use https://deno.land/x/outdent@v0.8.0 which seems to implement the tagged template to the same proposed spec to me!